src/cdk/drag-drop/drop.ts

```diff
@@ -30,6 +30,12 @@ import {moveItemInArray} from './drag-utils';
 /** Counter used to generate unique ids for drop zones. */
 let _uniqueIdCounter = 0;
 
+/**
+ * Buffer, in percent of the width/height, to add around the drop container when
```

Comment:
Need to update the description for the new name, something like
```
Proximity, as a ratio to width/height, at which a dragged item will affect the drop container.
```

Reply:
Done.